bs4.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "bs4",
```

Comment:
Please prefix python libraries with `python-`. It's not a mistake per se but I have a script that looks for commonly used modules.

Reply:
These are all generated files for the python deps

Reply:
Too bad. I'll submit pull request for this later.

Reply:
Yes, `flatpak-pip-generator` made those files for me.

Reply:
https://github.com/flatpak/flatpak-builder-tools/commit/47340df0c5a8625ae6a20ab1db96bdaa2f323105